security.tf

```diff
@@ -112,3 +112,10 @@ data "aws_iam_policy_document" "cross_org_instance_access" {
     sid = "ec2AssumeRoleCrossAccountStatementID"
   }
 }
+
+# SSM Policy
+resource "aws_iam_role_policy_attachment" "ec2_ssm_access" {
+  count      = var.enable_ssm_access == "" ? 1 : 0
```

Comment:
@fgauchat-binbash After you make the change I suggested (about changing the variable type to bool) you should be able to change this line to this:
```
count = var.enable_ssm_access ? 1 : 0
```